pkgs/by-name/ov/oversteer/package.nix

```diff
@@ -86,7 +88,9 @@ stdenv.mkDerivation {
 
   postInstall = ''
     substituteInPlace $out/lib/udev/rules.d/* \
-      --replace /bin/sh ${bash}/bin/sh
+      --replace-warn /bin/sh ${bash}/bin/sh
```

Comment:
I thought you said this one could be `--replace-fail`?

Reply:
Yes! Had a thought this was the suggested change, but I misread the diff.